packages/gallery/src/components/item/texts/lineHeightFixer.js

```diff
@@ -147,20 +147,21 @@ class LineHeightFixer {
         [PLACEMENTS.SHOW_ON_THE_LEFT]: 'gallery-item-left-info',
       }[styleParams.titlePlacement];
       const elements = itemContainer.getElementsByClassName(className);
-      const element = elements.length > 0 && elements[0];
-      const elementPadding =
-        parseInt(this.getCss(element, 'padding-top', 0)) +
-        parseInt(this.getCss(element, 'padding-bottom', 0));
-      const margin =
-        styleParams.imageInfoType === INFO_TYPE.SEPARATED_BACKGROUND &&
-        (styleParams.allowTitle || styleParams.allowDescription) &&
-        styleParams.textImageSpace
-          ? styleParams.textImageSpace
-          : 0;
-
-      const availableHeightOfTheTextElement = textPlacementAboveOrBelow ? options.externalTotalInfoHeight : textsContainerHeight; //maybe can be textsContainerHeight for textPlacementAboveOrBelow as well
-      availableHeight = availableHeightOfTheTextElement - elementPadding - margin;
-
+      if (element) {
```

Comment:
if (elements)

Reply:
I also think that the if is not needed. It needs to be tested, this is a very risky file to change stuff in.
Or maybe let the if to be closed before the 'const availableHeightOfTheTextElement'

Reply:
This can be removed in your refactor, it just broke the code